### PR TITLE
ci: fix packages and deploy

### PR DESCRIPTION
**Pull request by smola**
Signed-off-by: Santiago M. Mola <santi@mola.io>

### CODE REVIEW

**Comment by smola** [comment]
This should fix https://github.com/src-d/datasets/issues/54
I tested `make packages` locally, but Travis deploy is not tested yet.

File: Makefile [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 PROJECT = datasets
-COMMANDS = datasets/PublicGitArchive/borges-indexer/cmd/borges-indexer datasets/PublicGitArchive/multitool datasets/PublicGitArchive/pga
+COMMANDS = PublicGitArchive/borges-indexer/cmd/borges-indexer PublicGitArchive/multitool PublicGitArchive/pga
 
 # Including ci Makefile
 CI_REPOSITORY ?= https://github.com/src-d/ci.git
 CI_PATH ?= $(shell pwd)/.ci
 CI_VERSION ?= v1
-PKG_OS=linux darwin windows
+PKG_OS = linux darwin windows
```
Comment (campoy) [inline Makefile]: was that really the problem?

**Comment by smola** [comment]
@campoy Is it ok publishing a single tarball with pga, multitool and borges-indexer binaries? That's our default behavior for `make packages`, but we can keep previous behavior (tarball per binary) approach if needed.

**Comment by campoy** [comment]
It'd be fine ... but I think it's better to provide a way to download each binary separately to be honest.

**Comment by vmarkovtsev** [comment]
@smola @jfontan Shall we close this?

**Comment by jfontan** [comment]
I believe so. Windows is still not supported but there's an issue for that (#103)

**Pull request closed without merging**